src/test/java/com/google/javascript/clutz/partial/global_alias_missing.js

```diff
@@ -0,0 +1,6 @@
+goog.provide('ns.event');
+
+//!! Note that emitting just EventTarget would be wrong here
+//!! Assume there exists a ns.event.EventTarget.
```

Comment:
Is ns.event.EventTarget defined in another file?

Reply:
reworded the comment. PTAL